app/Http/Review/ReviewController.php

```diff
@@ -56,9 +56,9 @@ public function create()
      */
     public function store(Request $request)
     {
-        $this->reviews->create($request->all());
+        if($request) {$this->reviews->create($request->all());
```

Comment:
The `if` condition is not complete. It should check for a specific condition. For example, `if($request->isMethod('post'))`.